core/bin/snapshots_creator/src/creator.rs

```diff
@@ -266,6 +266,10 @@ impl SnapshotCreator {
         config: SnapshotsCreatorConfig,
         min_chunk_count: u64,
     ) -> anyhow::Result<()> {
+        tracing::info!(
+            "Starting snapshot creator with object store {:?} and config {config:?}",
+            self.blob_store
+        );
```

Comment:
unrelated question: what happens if we run creator on a pruned node?

Reply:
I haven't tested it, but theoretically it should work fine. There's nothing in the snapshot creator logic that requires complete node history.